server/settings.go

```diff
@@ -49,3 +49,42 @@ func (p *Plugin) settingsNotifications(header *model.CommandArgs, instanceID, ma
 
 	return p.responsef(header, "Settings updated. Notifications %s.", notifications)
 }
+
+func (p *Plugin) settingsWatching(header *model.CommandArgs, instanceID, mattermostUserID types.ID, connection *Connection, args []string) *model.CommandResponse {
+	const helpText = "`/jira watching watching [value]`\n* Invalid value. Accepted values are: `on` or `off`."
```

Comment:
```suggestion
	const helpText = "`/jira watching [value]`\n* Invalid value. Accepted values are: `on` or `off`."
```

Reply:
@jfrerich I've pushed a new commit for this change, could you please check it?